src/server/plugins/router.ts

```diff
@@ -11,6 +12,24 @@ export default {
     register: (server) => {
       server.route(routes)
       server.route([
+        {
+          method: 'get',
+          path: '/help/{slug?}',
```

Comment:
we have other pages underneath `/help/`. To avoid clashes with cookies, accessibility statement, etc., could we name this explicitly?

e.g. something like `/help/get-support/{slug?}`

Reply:
I'd seen this as the `index` or root page of the help section. I'll change it to `/help/get-support/{slug}` :+1:

Reply:
Resolved with d74a8b6f5e08c158f0277a96f3a712ca562d5830